api/src/main/java/pl/kolendateam/dadcard/characterCard/entity/Character.java

```diff
@@ -1,29 +1,158 @@
 package pl.kolendateam.dadcard.characterCard.entity;
 
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+import org.hibernate.annotations.JdbcTypeCode;
+import org.hibernate.type.SqlTypes;
+
 import jakarta.persistence.Entity;
 import jakarta.persistence.GeneratedValue;
 import jakarta.persistence.GenerationType;
 import jakarta.persistence.Id;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 import lombok.NonNull;
-import lombok.RequiredArgsConstructor;
 import lombok.Setter;
+import pl.kolendateam.dadcard.classCharacter.entity.ClassPg;
+import pl.kolendateam.dadcard.classCharacter.entity.SavingThrow;
+import pl.kolendateam.dadcard.skills.entity.ClassSkills;
+import pl.kolendateam.dadcard.skills.entity.Skills;
+
 
 @NoArgsConstructor
 @Getter
 @Setter
 @Entity
-@RequiredArgsConstructor
 public class Character {
 
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
-    int id;
+    public int id;
 
     @NonNull
     String characterName;
 
     @NonNull
     String playerName;
-}
+
+    @JdbcTypeCode(SqlTypes.JSON)
+    ArrayList<ClassPg> classPgArray;
+
+    int lep;
+
+    @JdbcTypeCode(SqlTypes.JSON)
+    SavingThrow savingThrow;
+
+    @JdbcTypeCode(SqlTypes.JSON)
+    ArrayList<ClassSkills> classSkills;
+
+    double skillPoints;
```

Comment:
skil point to liczba całkowita

Reply:
> skil point to liczba całkowita

Nie, może być częściowa, jak kupujesz na skillu który nie jest klasowy